controllers/recipeController.js

```diff
@@ -0,0 +1,39 @@
+import * as recipeService from "../services/recipeService.js";
+import HttpError from "../helpers/httpError.js";
+import ctrlWrapper from "../helpers/ctrlWrapper.js";
+
+const notFoundMessage = "Not found";
```

Comment:
It doesn't make sense to create a variable if it's used just once; just pass the string directly to HttpError